### PR TITLE
issue-20 [Documentation] Add missing existing feature parts to sections and reduce time for adoption by provide minimal example configurations

### PR DESCRIPTION
**Pull request by despot**
### Description

Thanks for contributing this Pull Request. Make sure that you submit this Pull Request against the `master` branch of this repository, add a brief description, and tag the relevant issue(s) and PR(s) below.

- Relevant Issues : (compulsory)
- Relevant PRs : (optional)
- Type of change :
  - [ ] New feature
  - [ ] Bug fix for existing feature
  - [ ] Code quality improvement
  - [ ] Addition or Improvement of tests
  - [x] Addition or Improvement of documentation

### CODE REVIEW

**Review by namitad — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Trapheus can be scheduled using a cloudwatch alarm rule or can be run using any
 │   └── restore_state_machine.png
 ├── src
 │   ├── checkstatus
-│   │   ├── DBClusterStatusWaiter.py              <-- Python Waiter for checking the status of the cluster
+│   │   ├── DBClusterStatusWaiter.py              <-- Python [Waiter](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/clients.html) for checking the status of the cluster
```
Comment (stationeros) [inline README.md:42]: Can we modify the link to https://boto3.amazonaws.com/v1/documentation/api/latest/guide/clients.html#waiters

Comment (despot) [reply README.md]: @stationeros thanks. Changed. Kindly review again?

Comment (stationeros) [reply README.md]: @despot  Thanks ... can you resolve the conflicting files.

Comment (despot) [reply README.md]: @stationeros done.

**Review by stationeros — APPROVED** [review]

**Pull request merged**